scripts/dev_scripts/copyright-checker.sh

```diff
@@ -22,7 +22,8 @@ for f in $files; do
         startyear=$currentyear
     fi
     if ! grep -i -e "Copyright (c) $startyear - $currentyear, Oracle and/or its affiliates. All rights reserved." "$f" 1>/dev/null;then
-        if [[ $f =~ .*\.(js$|py$|java$|tf$|go$|sh$|dl$|yaml$|gradle$|kts$|ini$|toml$) ]] || [[ "${f##*/}" = "Dockerfile" ]] || [[ "${f##*/}" = "Makefile" ]];then
+        if [[ $f =~ .*\.(js$|py$|java$|tf$|go$|sh$|dl$|yaml$|yml$|gradle$|kts$|ini$|toml$) ]] || [[ "${f##*/}" = "Dockerfile" ]] \
```

Comment:
I have just done some testing on this. It looks like ` "${f##*/}" = "Makefile"` and such will only match the files that has the name `Makefile`. Therefore, for files like `Makefile.prod` or `Dockerfile.prod`. This will not recognize and perform the copyright checking on them.

Reply:
> Makefile.prod

I don't think `Makefile.prod` is valid. For custom Dockerfile names the script needs more complex logic to detect them. I have created a separate [issue](https://github.com/oracle-samples/macaron/issues/188) for that.

Reply:
> I have just done some testing on this. It looks like ` "${f##*/}" = "Makefile"` and such will only match the files that has the name `Makefile`. Therefore, for files like `Makefile.prod` or `Dockerfile.prod`. This will not recognize and perform the copyright checking on them.

For custom file names, suchas Dockerfile names the script needs more complex logic to detect them. I have created a separate [issue](https://github.com/oracle-samples/macaron/issues/188) for that.